react/src/components/lablupTalkativotUI/LLMChatCard.tsx

```diff
@@ -127,6 +139,22 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
   const { token } = theme.useToken();
   const { t } = useTranslation();
 
+  useEffect(() => {
+    if (!_.isUndefined(inputMessage)) {
+      setInput(inputMessage);
+    }
+  }, [inputMessage, setInput]);
```

Comment:
How about adding a comment like this:
```suggestion
  // If the `inputMessage` prop exists, the `input` state has to follow it.
  useEffect(() => {
    if (!_.isUndefined(inputMessage)) {
      setInput(inputMessage);
    }
  }, [inputMessage, setInput]);
```